packages/ui-components/style/index.css

```diff
@@ -32,17 +32,12 @@ strong {
   font-weight: unset;
 }
 
-::selection {
-  /* background: unset; */
-}
-
 a {
   text-decoration: unset;
   color: unset;
 }
 
 a:hover {
-  cursor: unset;
```

Comment:
It looks like when it is implemented in browsers, we should really use `revert` instead of `unset` in these overrides. `revert` changes the property to the user-agent default, rather than the CSS spec default, which is what we want here. See https://developer.mozilla.org/en-US/docs/Web/CSS/revert

Reply:
I've never heard of `revert` before. It looks like Chrome (or Chrome devtools) doesn't support it?

Reply:
Right. Following the implementation bug links on mdn (click on the support boxes for each browser), you can see that Chrome and Firefox both don't support it and don't have anyone working on it right now.